### DISCUSSION TITLE
Calendar responisive to number of moons

### DISCUSSION THREAD

**Discussion by CiverBlack** [vigoren/foundryvtt-simple-calendar#259] [Q&A]
Hi,
so I have 4 moons in my calendar and it is a real struggle to see which moon icon belongs to which day. The calendar is allready responsive to the number of days in a week, would it be possible to make the width of the days responsive to the number of moons?
Also I would be helpfull if the new moons could have a colorindicator for the costum color I set in the settings.
Thanks

**Comment by vigoren** [comment]
This is something I am definitely looking into for calendars that have many moons.
